### PR TITLE
[UXE-2934] fix: adjustment max size component pick list

### PR DESCRIPTION
**Pull request by lucasmendes21**
### What is the new behavior introduced by this PR?
adjustment max size component pick list

### Does this PR introduce breaking changes?
- [ ] No
- [x] Yes 

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (fix, feat, test, etc)
- [ ] User inputs are sanitized/protected against malicious attacks
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Review by peterpaulo-azion — APPROVED** [review]

File: src/views/EdgeFirewall/FormFields/FormFieldsEdgeFirewall.vue [hunk 1]
```diff
@@ -108,7 +108,9 @@
           :pt="{
             sourceList: { class: [`h-80 ${classLoading}`] },
             targetList: { class: [`h-80 ${classLoading}`] },
-            buttons: { class: classLoading }
+            buttons: { class: classLoading },
+            sourceWrapper: { class: 'max-w-[340px]' },
+            targetWrapper: { class: 'max-w-[340px]' }
```
Comment (william-tome) [inline src/views/EdgeFirewall/FormFields/FormFieldsEdgeFirewall.vue:113]: Pergunta: isso vai funcionar para as telas grandes?

File: src/views/EdgeFirewall/FormFields/FormFieldsEdgeFirewall.vue [hunk 2]
```diff
@@ -117,10 +119,10 @@
           <template #targetheader>Selected</template>
           <template #item="slotProps">
             <div
-              class="flex flex-wrap p-2 align-items-center gap-3"
+              class="flex flex-wrap p-2 pl-0 align-items-center gap-3 max-w-xs"
```
Comment (william-tome) [inline src/views/EdgeFirewall/FormFields/FormFieldsEdgeFirewall.vue:122]: Mesma dúvida acima.

File: src/views/EdgeFirewall/FormFields/FormFieldsEdgeFirewall.vue (hunk 1 above)
Comment (lucasmendes21) [reply src/views/EdgeFirewall/FormFields/FormFieldsEdgeFirewall.vue:113]: Vai, sim, porque ele não passará desse tamanho.

File: src/views/EdgeFirewall/FormFields/FormFieldsEdgeFirewall.vue (hunk 2 above)
Comment (lucasmendes21) [reply src/views/EdgeFirewall/FormFields/FormFieldsEdgeFirewall.vue:122]: Vai, sim, porque ele não passará desse tamanho.

**Review by william-tome — APPROVED** [review]

**Pull request merged**